### PR TITLE
Add 2 missing members to Exception

### PR DESCRIPTION
**Pull request by danmosemsft**
Just needed the reflection change.

### CODE REVIEW

**Comment by danmosemsft** [comment]
@dotnet-bot test Innerloop Windows_NT Debug Build and Test please (failure in System.Net.Http.Functional.Tests.PostScenarioTest.PostUsingNoSpecifiedSemantics_UsesChunkedSemantics)

**Comment by danmosemsft** [comment]
@dotnet-bot test innerloop ubuntu14.04 release build and test (build hang after System.Security.Claims.Tests)

**Comment by danmosemsft** [comment]
@dotnet-bot test innerloop centos7.1 debug build and test (System.Net.Http.Functional.Tests.HttpClientHandler_DefaultProxyCredentials_Test.ProxyExplicitlyProvided_DefaultCredentials_Ignored failed)

**Comment by danmosemsft** [comment]
@dotnet-bot test Innerloop Windows_NT Release Build and Test System.Net.Http.Functional.Tests.HttpClientHandler_DefaultProxyCredentials_Test.ProxyExplicitlyProvided_DefaultCredentials_Ignored failed)

**Comment by danmosemsft** [comment]
@stephentoub  ok?

File: src/System.Runtime/tests/System/ExceptionTests.cs [hunk 1]
```diff
@@ -4,69 +4,37 @@
...
-            Assert.Equal(i.InnerException.Message, innerExceptionMessage);
-            Assert.Equal(i.InnerException.HResult, ex.HResult);
-            Assert.Equal(E_POINTER, (uint)i.HResult);
+            Assert.Equal(typeof(Exception), (new Exception()).GetType());
+            Assert.Equal(typeof(NullReferenceException), (new NullReferenceException()).GetType());
         }
 
         [Fact]
-        public static void EndOfStreamException_Ctor_Empty()
+        public static void Exception_TargetSite()
```
Comment (stephentoub) [inline src/System.Runtime/tests/System/ExceptionTests.cs]: In other projects, we've separated out the tests for netstandard1.7... have we done that with System.Runtime?

**Comment by stephentoub** [comment]
LGTM

**Comment by danmosemsft** [comment]
The tests are broken out for 1.7, yes.

**Pull request merged**